src/org/traccar/model/Position.java

```diff
@@ -65,15 +66,45 @@
     public static final String PREFIX_ADC = "adc";
     public static final String PREFIX_IO = "io";
     public static final String PREFIX_COUNT = "count";
+    public static final String PREFIX_BUTTON = "button";
 
+    public static final String ENDING_PRESSED = "Pressed";
```

Comment:
I think all this (`ENDING_*`) should be just info. Possibly we can ignore it for now because most likely IO status is already saved in some other attribute.